ldap/servers/slapd/auth.c

```diff
@@ -505,18 +576,38 @@ handle_handshake_done(PRFileDesc *prfd, void *clientData)
         slapi_sdn_free(&sdn);
         (void)slapi_getSSLVersion_str(channelInfo.protocolVersion,
                                       sslversion, sizeof(sslversion));
-        slapi_log_access(LDAP_DEBUG_STATS,
-                         "conn=%" PRIu64 " %s client bound as %s\n",
-                         conn->c_connid,
-                         sslversion, clientDN);
+        if (log_format != LOG_FORMAT_DEFAULT) {
+            slapd_log_pblock_init(&logpb, log_format, NULL);
+            logpb.conn_id = conn->c_connid;
+            logpb.msg = "client bound";
+            logpb.tls_version = sslversion;
+            logpb.client_dn = clientDN;
+            slapd_log_access_tls_client_auth(&logpb);
+        } else {
+            slapi_log_access(LDAP_DEBUG_STATS,
+                            "conn=%" PRIu64 " %s client bound as %s\n",
+                            conn->c_connid,
+                            sslversion, clientDN);
+        }
     } else if (clientCert != NULL) {
         (void)slapi_getSSLVersion_str(channelInfo.protocolVersion,
                                       sslversion, sizeof(sslversion));
-        slapi_log_access(LDAP_DEBUG_STATS,
-                         "conn=%" PRIu64 " %s failed to map client "
-                         "certificate to LDAP DN (%s)\n",
-                         conn->c_connid,
-                         sslversion, extraErrorMsg);
+        if (log_format != LOG_FORMAT_DEFAULT) {
+            slapd_log_pblock_init(&logpb, log_format, NULL);
+            logpb.conn_id = conn->c_connid;
+            logpb.msg = "failed to map client certificate to LDAP DN";
+            logpb.tls_version = sslversion;
+            logpb.client_dn = clientDN;
```

Comment:
Minor nitpick, clientDN is NULL here, so it'll be more explicit to set it so directly. But I don't have a hard preference here... 
```c
            logpb.client_dn = NULL;